contracts/bridge/interfaces/IMiniChefV2.sol

```diff
@@ -8,7 +8,7 @@ import {IRewarder} from "./IRewarder.sol";
 interface IMiniChefV2 {
     struct UserInfo {
         uint256 amount;
-        uint256 rewardDebt;
+        int256 rewardDebt;
```

Comment:
Issue here is now this no longer matches the canonical minichef2 (sushi + https://github.com/synapsecns/synapse-contracts/blob/master/contracts/bridge/MiniChefV2.sol)

Reply:
From that link:

MiniChefV2: 

```
    struct UserInfo {
        uint256 amount;
        int256 rewardDebt;
    }
```

MiniChefV21.sol/IMiniChefV2:

```
    struct UserInfo {
        uint256 amount;
        int256 rewardDebt;
    }
```

Reply:
Canonical Sushi (https://github.com/sushiswap/sushiswap/blob/master/contracts/MiniChefV2.sol)

```
    struct UserInfo {
        uint256 amount;
        int256 rewardDebt;
    }
```

Reply:
For whatever reason, the original struct in the interface had the non-canonical signature.